Rakefile

```diff
@@ -21,14 +23,17 @@ require 'bundler_api/gem_helper'
 $stdout.sync = true
 Thread.abort_on_exception = true
 
+RuboCop::RakeTask.new
+
 begin
   require 'rspec/core/rake_task'
 
   desc "Run specs"
   RSpec::Core::RakeTask.new(:spec) do |t|
     t.rspec_opts = %w(-fs --color)
-    #t.ruby_opts  = %w(-w)
+    Rake::Task['rubocop'].invoke
```

Comment:
To be completely honest, this was the first Rake task I have created. And it is running RuboCop and the tests when you run `rake spec`, as you asked. But when I put the call outside the block, it also works.

What's the proper way to do it?